flatpak/org.nanuc.Axolotl.yml

```diff
@@ -52,11 +52,9 @@ modules:
     buildsystem: simple
     build-options:
       # Add the node bin directory.
-      append-path: "/usr/lib/sdk/node18/bin:/run/build/axolotl/flatpak-node/vite:/usr/lib/sdk/rust-stable/bin"
+      append-path: /usr/lib/sdk/node18/bin:/usr/lib/sdk/rust-stable/bin
       env:
-        # Set the cache directory, used to find Electron and some other helper
-        # tools.
-        # (The directory format is: /run/build/MODULE_NAME/flatpak-node/cache)
+        # Set the cache directory
         XDG_CACHE_HOME: /run/build/axolotl/flatpak-node/cache
```

Comment:
Do we need this
```suggestion
```

Reply:
Honestly, I dont think so. Wasnt sure what a good scope was for this PR, but lets add some scope creep and remove these two configs (assuming build still holds)